impeller/display_list/aiks_dl_vertices_unittests.cc

```diff
@@ -468,5 +468,54 @@ TEST_P(AiksTest, DrawVerticesTextureCoordinatesWithFragmentShader) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+// The vertices should form a solid red rectangle with no gaps.
+// The blue rectangle drawn under them should not be visible.
+TEST_P(AiksTest,
+       DrawVerticesTextureCoordinatesWithFragmentShaderNonZeroOrigin) {
+  std::vector<SkPoint> positions_lt = {
+      SkPoint::Make(200, 200),  //
+      SkPoint::Make(250, 200),  //
+      SkPoint::Make(200, 250),  //
+      SkPoint::Make(250, 250),  //
+  };
+
+  auto vertices_lt = flutter::DlVertices::Make(
+      flutter::DlVertexMode::kTriangleStrip, positions_lt.size(),
+      positions_lt.data(),
+      /*texture_coordinates=*/positions_lt.data(), /*colors=*/nullptr,
+      /*index_count=*/0,
+      /*indices=*/nullptr);
+
+  flutter::DisplayListBuilder builder;
+  flutter::DlPaint paint;
+  flutter::DlPaint rect_paint;
+  rect_paint.setColor(DlColor::kBlue());
+
+  auto runtime_stages =
+      OpenAssetAsRuntimeStage("runtime_stage_position.frag.iplr");
+
+  auto runtime_stage =
+      runtime_stages[PlaygroundBackendToRuntimeStageBackend(GetBackend())];
+  ASSERT_TRUE(runtime_stage);
+
+  auto runtime_effect = DlRuntimeEffect::MakeImpeller(runtime_stage);
+  auto rect_data = std::vector<Rect>{Rect::MakeLTRB(200, 200, 250, 250)};
+
+  auto uniform_data = std::make_shared<std::vector<uint8_t>>();
+  uniform_data->resize(rect_data.size() * sizeof(Rect));
+  memcpy(uniform_data->data(), rect_data.data(), uniform_data->size());
+
+  auto color_source = flutter::DlColorSource::MakeRuntimeEffect(
+      runtime_effect, {}, uniform_data);
+
+  paint.setColorSource(color_source);
+
+  builder.Scale(GetContentScale().x, GetContentScale().y);
+  builder.DrawRect(SkRect::MakeLTRB(200, 200, 250, 250), rect_paint);
+  builder.DrawVertices(vertices_lt, flutter::DlBlendMode::kSrcOver, paint);
+
+  ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
```

Comment:
Heads up you are going to need to add the included golden file to the list.

Reply:
I usually just wait for the check to fail first and then copy the string.

Reply:
done